success-criteria.md

```diff
@@ -113,34 +113,19 @@ behind this decision ought to be thoroughly documented.
 
 This document considers web advertising from three [points of
 view](https://www.w3.org/Consortium/Points/)[POINTS]: that of individual web
-users (both in aggregate and individually), publishers and their partners (both
+users (both in aggregate and individually), --publishers and their partners (both
 authors and the business model that funds them) and the delivery access
-mechanism (both connectivity and navigation).
-
-Any technology can be abused. Open societies do not attempt to suppress
-technology, but rather put [appropriate
-regulations](https://iabeurope.eu/all-news/iab-europes-press-statement-openrtb-and-eu-data-protection-law)
-[OPEN-RTB-EU-LAW] in place to define acceptable and unacceptable uses of that
-technology. For example, automobiles are not required to integrate functionality
-that technically prevents them from exceeding the speed limit. Instead, drivers
-are educated and trained in traffic rules, and drivers who violate speed limits
-are subject to fines and/or deprived of their permits. However, documenting
-specific criteria as to what constitutes a violation helps enable easier
-detection and reporting of non-compliance with the regulations that govern
-technology.
+mechanism (both connectivity and navigation).--
```

Comment:
Most publishers would, I suspect, find points of difference from ad technology companies in concerns and descriptions of success. For example the two parties have  different requirements in their interactions with users and see different regulatory issues in legal frameworks. I think we need to revise this representation as it stands here and throughout. We should aim for a more appropriate separation of concerns to resolve some of the issues privacy, both as a technical and legal challenge, brings forth. I’d like feedback from the authors about how we can rephrase this section to make those changes clearer.

Reply:
Much agreed. Success criteria between publishers and adtech have historically not be in full alignment and conflating them won't bring clarity to the discussion. Success criteria for each should be produced separately, and then the community can decide what to focus on in terms of priorities and which to favour in case of conflict. Adtech is not represented in the [Priority of Constituencies](https://www.w3.org/TR/html-design-principles/#priority-of-constituencies), which I would say is a bug.

Reply:
> JMK: Aram correctly points out that publihsers and ad tech companies' interests are not always aligned and hence their concerns and requirements should be clearly called out, so developers can understand how their specifications may impact them.

> However, this document does attempt to distinguish their distinct concerns and requirements. I would more granular feedback as to which interests or concerns are not yet included but should be.

My response:

I agree with @darobin here, these two interests need to be clearly delineated overall in order to find a solution. Otherwise we are essentially tricking ourselves into solutions that will degrade both relationships and interests between both parties down the line. Clear delineated constituencies are one of the things my edits attempt to bring to this document overall. 

I do not have an edit given here, but I highlighted this specific phrasing because it definitely implies that these interests are the same. (emphasis mine)

> This document considers web advertising from three points of view[POINTS]: that of individual web users (both in aggregate and individually), **publishers and their partners (both authors and the business model that funds them)** and the delivery access mechanism (both connectivity and navigation). 

makes it seem to me that the document intends to bucket interests as follows:

- web users
- publishers and their partners
- deliver access mechanisms

as indicated by the phrase 'three points of view'. I disagree with the idea that these represent three coherent singular points of view. There are arguably at least four (separating publishers from ad tech) but could be more than that. I'm not sure if it even makes sense to try and represent all four of those views within a single document, because in some cases those points of view may differ radically (an ad exchange, for example, prioritizes scale to an extent that drives their interests in a far different way than a local paper). As noted by @darobin it might even make sense, because these interests differ, to split them into different documents for clarity's sake. (Especially considering that one could argue that even publishers on their own do not constitute a monolithic single view, though that is an argument for another time.) But even if we do not do that, we need to make it clear up top that the publisher and ad tech concerns are not a singular coherent point of view. And developer changes will impact those concerns very differently which we would want to create feedback around and a conversation for such changes that would allow participants to weigh and understand those different impacts to different parties and interests.

Reply:
The document updated yesterday starts to break out these differences. I think more work is needed and would like to setup a conference call to resolve this point across multiple contributors.